functions/zoom-on-demand.py

```diff
@@ -153,4 +168,13 @@ def handler(event, context):
         )
         return resp(500, err_msg)
 
+    set_pipeline_status(
+        request_id,
+        PipelineStatus.ON_DEMAND_RECEIVED,
+        meeting_id=webhook_data["payload"]["object"]["id"],
+        recording_id=recording_uuid,
+        recording_start_time=webhook_data["payload"]["object"]["start_time"],
+        topic=webhook_data["payload"]["object"]["topic"],
+        origin="on_demand",
+    )
```

Comment:
I guess my thinking on this was that the requestor would know that their request failed because they would see an error in Opencast. Otherwise you might see a lot of failed requests in the status tools like Slack, etc. But maybe we want that data?